pkg/node/node.go

```diff
@@ -195,6 +224,13 @@ func (s *Service) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeR
 		return nil, status.Error(codes.InvalidArgument, "staging target path is required")
 	}
 
+	nvmeIP := strings.Split(req.PublishContext["PORTAL0"], ":")
```

Comment:
can we declare "PORTAL0", ":4420" and similar strings as const and use it?

Reply:
Since there is no rest API call for getting the initiators on the array side, the only way is using the gonvme library, so in order to get the IP, had to do this step